_docs-sources/intro/overview/business-case.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
I question whether this belongs on the docs site. I understand we're releasing this to the general public, but what's the line between marketing and docs? Seems like this is making the kinds of points that belong on our main site and that docs should be reserved for content related specifically to using our product. I understand that we can have content that builds general knowledge or is more like "building the mental model" type stuff - but this seems more then that